fe/user/src/pages/MapPage/Panel/VenueDetail/ShowDetail/index.tsx

```diff
@@ -8,25 +8,47 @@ import { useOutletContext } from 'react-router-dom';
 import SwiperCore from 'swiper';
 import { Navigation } from 'swiper/modules';
 import { Swiper, SwiperSlide } from 'swiper/react';
+import { useShallow } from 'zustand/react/shallow';
+import { useShowDetailStore } from '~/stores/useShowDetailStore';
 import { ShowDetail as ShowDetailData } from '~/types/api.types';
 
 type Props = {
   showList: ShowDetailData[];
-  currentIndex: number;
-  onCloseClick: () => void;
 };
 
-export const ShowDetail: React.FC<Props> = ({
-  showList,
-  currentIndex,
-  onCloseClick,
-}) => {
+export const ShowDetail: React.FC<Props> = ({ showList }) => {
   const mapElement = useOutletContext<React.RefObject<HTMLDivElement>>();
   const [swiper, setSwiper] = useState<SwiperCore>();
+  const { showDetailId, setShowDetailId } = useShowDetailStore(
+    useShallow((state) => ({
+      showDetailId: state.showDetailId,
+      setShowDetailId: state.setShowDetailId,
+    })),
+  );
+
+  const currentSwiperIndex = showList.findIndex(
+    (show) => show.id === showDetailId,
+  );
+
+  const onClickPrevButton = () => {
+    const prevIndex = currentSwiperIndex - 1 < 0 ? 0 : currentSwiperIndex - 1;
```

Comment:
`0보다 작으면` 0, 아니면 `currentSwiperIndex - 1`을 리턴하는 조건문인 것 같네요 
그렇다면 다르게 표현하면 둘중 큰 것을 리턴하는 조건문이 되는데 아래 처럼 수정하면 조금 더 간결하긴 하게 표현이 되긴 할 것 같네요
```suggestion
    const prevIndex = Math.max(currentSwiperIndex - 1, 0);
```

Reply:
간결하고 좋아보입니다 👍👍

Reply:
캬 좋습니다! 따봉 드려요 👍

Reply:
수정 완료 https://github.com/jazz-meet/jazz-meet/pull/289/commits/3f766c96238cc45cd3e6b70aa310e2d2cfb7a7ae